src/features/pro/components/ProBanner.tsx

```
@@ -85,9 +85,9 @@ const NotProUser = ({ totalEarnings }: { totalEarnings: number }) => {
 const NotLoggedInUser = () => {
   return (
     <div className="z-20 w-72">
-      <p className="text-2xl font-medium text-white md:text-3xl md:font-semibold">
-        You’re very close to being eligible
-      </p>
+      <h1 className="whitespace-nowrap text-2xl font-medium text-white md:text-3xl md:font-semibold">
```

ESLint

[error] Replace `whitespace-nowrap·text-2xl·font-medium` with `text-2xl·font-medium·whitespace-nowrap`

(prettier/prettier)